pandas/tests/indexing/test_indexing.py

```diff
@@ -1202,3 +1202,12 @@ def test_readonly_indices():
     result = df["data"].iloc[indices]
     expected = df["data"].loc[[1, 3, 6]]
     tm.assert_series_equal(result, expected)
+
+
+def test_1tuple_without_multiindex():
```

Comment:
This works already on master right? So not a behavior change just additional coverage?

Reply:
Correct.  This adds coverage for the case where the exception on L1134 is reached